images/onboarding/wcpay.svg

```diff
@@ -0,0 +1,26 @@
+<svg
+	width="100"
+	height="64"
+	viewBox="-10 0 120 64"
+	fill="none"
+	xmlns="http://www.w3.org/2000/svg"
+>
+	<path
+		fillRule="evenodd"
```

Comment:
I think this needs to be updated to `fill-rule` and same for `clip-rule`.

Reply:
Ah, good catch.